### ISSUE TITLE
Periodic task docs

### ISSUE THREAD

**Issue by mindojo-victor** [coleifer/huey#250]
I have this code:
```python
from huey.contrib.sqlitedb import SqliteHuey
from huey import crontab

huey = SqliteHuey('my_app', filename='huey.db')

@huey.periodic_task(crontab(minute='*'))
def count_beans(num):
    print('-- counted %s beans --' % num)
```
I one terminal I started `huey_consumer.py tasks.huey --worker-type=process` and in another `huey_consumer.py tasks.huey --worker-type=process --no-periodic`

http://huey.readthedocs.io/en/latest/api.html#Huey.periodic_task

> By default, the consumer will execute `periodic_task` functions. To disable this, run the consumer with `-n` or `--no-periodic`.

From what I see this is not true. 
First terminal
```
[2017-10-04 14:48:52,657] INFO:huey.consumer.Scheduler:35768:Scheduling periodic task queuecmd_count_beans: queuecmd_count_beans.
[2017-10-04 14:48:59,135] INFO:huey.consumer.Worker:35769:Executing queuecmd_count_beans: queuecmd_count_beans
-- counted 100 beans --
[2017-10-04 14:49:52,713] INFO:huey.consumer.Scheduler:35768:Scheduling periodic task queuecmd_count_beans: queuecmd_count_beans.
[2017-10-04 14:49:56,913] INFO:huey.consumer.Worker:35769:Executing queuecmd_count_beans: queuecmd_count_beans
-- counted 100 beans --
[2017-10-04 14:50:52,775] INFO:huey.consumer.Scheduler:35768:Scheduling periodic task queuecmd_count_beans: queuecmd_count_beans.
[2017-10-04 14:50:54,697] INFO:huey.consumer.Worker:35769:Executing queuecmd_count_beans: queuecmd_count_beans
-- counted 100 beans --
[2017-10-04 14:51:52,831] INFO:huey.consumer.Scheduler:35768:Scheduling periodic task queuecmd_count_beans: queuecmd_count_beans.
[2017-10-04 14:52:52,896] INFO:huey.consumer.Scheduler:35768:Scheduling periodic task queuecmd_count_beans: queuecmd_count_beans.
[2017-10-04 14:53:52,952] INFO:huey.consumer.Scheduler:35768:Scheduling periodic task queuecmd_count_beans: queuecmd_count_beans.
[2017-10-04 14:54:53,006] INFO:huey.consumer.Scheduler:35768:Scheduling periodic task queuecmd_count_beans: queuecmd_count_beans.
[2017-10-04 14:55:53,068] INFO:huey.consumer.Scheduler:35768:Scheduling periodic task queuecmd_count_beans: queuecmd_count_beans.
```
Second terminal
```
[2017-10-04 14:51:59,897] INFO:huey.consumer.Worker:35772:Executing queuecmd_count_beans: queuecmd_count_beans
-- counted 100 beans --
[2017-10-04 14:52:57,681] INFO:huey.consumer.Worker:35772:Executing queuecmd_count_beans: queuecmd_count_beans
-- counted 100 beans --
[2017-10-04 14:53:55,456] INFO:huey.consumer.Worker:35772:Executing queuecmd_count_beans: queuecmd_count_beans
-- counted 100 beans --
[2017-10-04 14:54:53,235] INFO:huey.consumer.Worker:35772:Executing queuecmd_count_beans: queuecmd_count_beans
-- counted 100 beans --
[2017-10-04 14:55:59,770] INFO:huey.consumer.Worker:35772:Executing queuecmd_count_beans: queuecmd_count_beans
-- counted 100 beans --
```

So `--no-periodic` disables the scheduler, while executing the tasks scheduled by a scheduler from another consumer.

Is this correct? Are the docs wrong?

**Comment by mindojo-victor** [comment]
If so, I think it's worth mentioning in the docs that this option is need to prevent duplicate scheduling when there are several consumers in different process -- each of them tries to schedule the periodic task in the specified times.

**Comment by coleifer** [comment]
The tasks are being enqueued by the first consumer and executed by a worker in the second. This is by design. It allows you to run multiple consumers reading from a single queue if you so desire (and is a bit of a holdover from before huey consumer got multiprocessing support).

Let's say you had 2 huey instances, instantiated with a unique `name` parameter and with `global_registry=False` (meaning a task decorated by one huey instance is not visible to the 2nd huey instance). If you started up two consumers, the two would not interact with each other because they are separate instances using separate queues.

In other words, the no-periodic option does not restrict which tasks a consumer will run. It only restricts whether or not that consumer process will run a scheduler which enqueues periodic tasks.

**Comment by mindojo-victor** [comment]
> In other words, the no-periodic option does not restrict which tasks a consumer will run.

That's my point and the issue. The docs should be changed, because right now the can be read wrongly:

> By default, the consumer will execute `periodic_task` functions. To disable this, run the consumer with `-n` or `--no-periodic`.

**Comment by coleifer** [comment]
Updated the docs, the wording was indeed ambiguous. Thanks!
